core/chains/evm/gas/rollups/l1_oracle.go

```diff
@@ -49,10 +54,11 @@ func IsRollupWithL1Support(chainType chaintype.ChainType) bool {
 	return slices.Contains(supportedChainTypes, chainType)
 }
 
-func NewL1GasOracle(lggr logger.Logger, ethClient l1OracleClient, chainType chaintype.ChainType, daOracle evmconfig.DAOracle) (L1Oracle, error) {
+func NewL1GasOracle(lggr logger.Logger, ethClient l1OracleClient, chainType chaintype.ChainType, daOracle evmconfig.DAOracle, clientsByChainID map[string]Client) (L1Oracle, error) {
```

Comment:
doesn't look like `clientsByChainID`  is being used... is that a mistake or should we do the following

```suggestion
func NewL1GasOracle(lggr logger.Logger, ethClient l1OracleClient, chainType chaintype.ChainType, daOracle evmconfig.DAOracle, _ map[string]Client) (L1Oracle, error) {
```

Reply:
That's right, currently `clientsByChainID ` is not used, it's just made available for CCIP to make heuristic based fee calculation for future L2 like Linea. I'm okay with change it to `_`, but it might be easier for Oliver to add his work later.

Reply:
resolved in telecom